api/client/notifications_v1_alpha.go

```diff
@@ -24,8 +24,18 @@ func NewNotificationsV1AlphaApi() NotificationsV1AlphaApi {
 	}
 }
 
-func (c *NotificationsV1AlphaApi) ListNotifications() (*models.NotificationListV1Alpha, error) {
-	body, status, err := c.BaseClient.List(c.ResourceNamePlural)
+func (c *NotificationsV1AlphaApi) ListNotifications(pageSize int32, pageToken string) (*models.NotificationListV1Alpha, error) {
+	query := ""
```

Comment:
Building the query here would become simpler if we used the `net/url` package - see [this one](https://github.com/semaphoreci/cli/blob/master/api/client/agent_v1_alpha.go#L28) for example

Reply:
@lucaspin done, please check again